### DISCUSSION TITLE
HomeKit for controlling a DMX light

### DISCUSSION THREAD

**Discussion by Fibbefobbe** [node-dmx/dmx#136] [General]
Hi to all experts,

I have a pool in the garden, that has a DMX light inside. I can control the light ("rgbw") with a device called "air2dmx" (https://usb2dmx.com/produkte/air2dmx). 
Now, the light has an IP in my network.

I want to control the light with homekit. I tried very much, but I cannot get a working solution. The best way was to try a plugin for homebridge and I was able to get a switch in HomeKit, but the switch was not working.

Can anybody help an enthusiastic but unsuccessful man???

**Comment by hrueger** [comment]
Hi @Fibbefobbe,
I don't know anything about Homekit nor homebridge, but by googleing I suggest that you try something like this https://www.npmjs.com/package/homebridge-artnet as the air2dmx Interface is just an Artnet receiver.

**Reply by Fibbefobbe** [reply]
Hi,
I tried my very best to fill in the right data and I was able to see a new button in HomeKit ... but it did not work. The light was not reacting 😐

**Reply by hrueger** [reply]
Then you should probably reach out to the maintainer of the homebridge plugin, because we just create the library here... Sorry for that.

**Comment by wiedi** [comment]
There is also https://github.com/bluemaex/homebridge-dmxuniverse (not sure if that was what you tried)

**Reply by Fibbefobbe** [reply]
I have the problem, that I don't know how to use the plugIn. I have the dms art net (air2dmx), I have the IP address and the port. 
What is the difference between "bridge" and "host"? 

🫠🫠
